strider/server.py

```diff
@@ -50,7 +50,7 @@
     title="Strider",
     description=DESCRIPTION,
     docs_url=None,
-    version="3.18.4",
+    version="4.0.1",
```

Comment:
Do we need to bump the version?

Reply:
Good catch. I was thinking we don't need to bump the version, but I forgot that we had made changes to main since the last release.